src/theme/deployment/components/_display-group.scss

```diff
@@ -50,11 +50,18 @@ plh-tmpl-display-group .display-group-wrapper[data-param-style~="parent_point"]
     max-width: 160px;
     padding: 16px;
   }
-  &[data-param-style~="tight"] {
+  @media (max-width: 375px) {
     plh-template-component {
-      display: block;
-      max-height: 70px;
-      max-width: 160px;
+      padding: 4px;
+    }
+  }
+  &[data-param-style~="grid-spacing-sm"] {
+    grid-gap: 16px;
```

Comment:
looks good to me, little chance of unintended knock-ons when using selectors in this way.